source/unsafe-messenger.js

```diff
@@ -73,6 +75,35 @@ export default function unsafeMessenger() {
 			editor.blur();
 		});
 	}
+
+	function monacoEditor(target) {
+		const editor = monaco.editor.getModels()[0];
```

Comment:
There can be multiple fields on the page, this would only work on the first one, even if you highlight the last one